src/http/ngx_http.c

```diff
@@ -1845,6 +1845,7 @@ ngx_http_add_listening(ngx_conf_t *cf, ngx_http_conf_addr_t *addr)
 #endif
```

Comment:
commit: I think you should change:

> This patch and is sister commits add the `mptcp` parameter to the `listen` directive.

and have it specific per commit:

    This patch adds a new parameter 'mptcp' to the  'listen' directive
    in the HTTP module.

Also, you can do the same modification for the titles as mentioned in your first commit:

     HTTP: added MPTCP support.

And I think you can add something like:

    Even if MPTCP is supported by different OS, only Linux supports the
    `IPPROTO_MPTCP` protocol, which is why this feature is currently
    limited to Linux only.


Same in the two next commits of course.